task/net/src/main.rs

```diff
@@ -10,15 +10,32 @@ mod buf;
 mod server;
 
 use core::sync::atomic::{AtomicU32, Ordering};
-use stm32h7::stm32h743 as device;
-
+cfg_if::cfg_if! {
+    if #[cfg(any(target_board = "nucleo-h743zi2", target_board = "sidecar-1"))] {
+    use stm32h7::stm32h743 as device;
```

Comment:
rustfmt doesn't fix macros every time, unfortunately -- please indent the bodies of these curly braced blocks.